src/components/UI/Search.tsx

```diff
@@ -0,0 +1,19 @@
+import { DocSearch } from '@docsearch/react';
+
+import '@docsearch/css';
+
+export const Search: React.FC = () => {
+  const appId = process.env.NEXT_PUBLIC_ALGOLIA_APP_ID || '';
+  const apiKey = process.env.NEXT_PUBLIC_ALGOLIA_SEARCH_API_KEY || '';
+  const indexName = process.env.NEXT_PUBLIC_ALGOLIA_BASE_SEARCH_INDEX_NAME || '';
+
+  // TODO: Replace Algolia test keys with above env vars when ready
+  return (
+    <DocSearch appId={appId} apiKey={apiKey} indexName={indexName} transformItems={
+      items => items.map(item => ({
+        ...item,
+        url: item.url.replace('https://melodious-puffpuff-8e1109.netlify.app/', ''),
```

Comment:
```suggestion
        url: item.url.replace('https://melodious-puffpuff-8e1109.netlify.app/, ''),
```
I suspect we want to keep the trailing slash

Reply:
```suggestion
        url: item.url.replace('https://melodious-puffpuff-8e1109.netlify.app', ''),
```

Reply:
(that's what we do here: https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/Search/index.tsx#L123)

Reply:
Hm, good call, let's see how that does.

Reply:
Hm, still seeing the same bug where the routes are stacking after removing the trailing slash.

Reply:
Actually, I may have spoke too soon. I think I was having issues with cookies/localStorage not updating when I made this change and tested, and it was using the old links. I switched to a private browser window and am having better luck. 🤞 Hopefully the Netlify preview will pass this time and we can try it out on that.

Reply:
I think that fixed the issues of the URLs stacking, thanks!